reconcile/utils/saasherder/saasherder.py

```diff
@@ -982,6 +983,7 @@ def _process_template(self, spec: TargetSpec) -> tuple[list[Any], Promotion | No
                 ref=ref,
                 namespace=spec.target.namespace.name,
                 values=consolidated_parameters,
+                api_versions=spec.oc.api_resources or [],
```

Comment:
need to do convert `api_resources` (`dict[str, list[OCCliApiResource]]`) to deduplicated `list[str]`